tests/security/scap_workbench/scap_workbench.pm

```diff
@@ -34,10 +34,12 @@ sub run {
     enter_cmd('scap-workbench &');
     assert_screen('scap-workbench-started', timeout => 120);
 
-    # Load 'Sle15' content
-    record_info('Load Sle15');
-    assert_and_click('scap-workbench-Select-content-to-load');
-    assert_and_click('scap-workbench-Sle15');
+    # Load 'Sle15' content if the system is sle
```

Comment:
Actually, we always use SLE15 instead of Sle15 since it is the abbreviation of **S**USE **L**inux **E**nterprise 15. But it is not a big deal in the comment.